inventory/sample.aws.example.com.d/inventory/ec2.ini

```diff
@@ -0,0 +1,190 @@
+#{{ ansible_managed | Ansible EC2 external inventory script settings }}
```

Comment:
Same comments as above

Reply:
Yes, basically HA is an extension of non-HA inventory so the typos will be the same. Will review both